function testFunction() {
    error? errorCause = error("error cause");
    error newError = error("error name", );
}
